internal/scti/ctlog_test.go

```diff
@@ -3,21 +3,33 @@ package scti
 import (
 	"context"
 	"crypto"
+	"crypto/ecdsa"
+	"crypto/ed25519"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"os"
 	"strings"
 	"testing"
 
-	"github.com/google/trillian/crypto/keys/pem"
+	trillianpem "github.com/google/trillian/crypto/keys/pem"
```

Comment:
I thought you were trying to remove dependencies on trillian from TesseraCT?

Reply:
Correct. This pull request needs the "encoding/pem" module. I will remove the "github.com/google/trillian/crypto/keys/pem" dependency in the another pull request. Created https://github.com/transparency-dev/static-ct/issues/183 to track.